src/qrcode/utils.ts

```diff
@@ -47,7 +47,7 @@ export enum QRCodeTypes {
 
 const TAG = 'QR/utils'
 
-const QRFileName = '/celo-qr.png'
+const QRFileName = '/valora-qr.png'
```

Comment:
https://www.figma.com/file/RsO1zYTuQZbClu9zjxOKe5/Celo-Audit%2FQR?type=design&node-id=1-3&mode=design&t=05QbrimCDRAGoLqL-0 seems like we want valora_address-qr.png 
```suggestion
const QRFileName = '/valora_address-qr.png'
```